src/main/java/io/anserini/collection/TwentynewsgroupsCollection.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Anserini: A Lucene toolkit for replicable information retrieval research
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.anserini.collection;
+
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.io.FileReader;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+/**
+ * An instance of the <a href="http://people.csail.mit.edu/jrennie/20Newsgroups">20 Newsgroup by date</a>.
+ * The collection contains 18846 documents, 20 different news sets,each corresponding to a different topic. 
+ * stored in plain text format. The collection is 14MB compressed, 35.9MB uncompressed.
+ */
+public class TwentynewsgroupsCollection extends DocumentCollection<TwentynewsgroupsCollection.Document> {
```

Comment:
Rename to `20Newsgroups`?

Reply:
Emmm, I don't think 20Newsgroup is a valid name. I tried it before, but it doesn't allow me to.

Reply:
Ah, I see!